projects/batfish/src/test/resources/org/batfish/grammar/cisco/testrigs/ibgp-reject-routerid-match/configs/r1

```diff
@@ -0,0 +1,22 @@
+! boot system flash:EOS.swi
+hostname r1
+interface Ethernet1
+   no switchport
+   ip address 10.14.0.1/24
+!
+interface Loopback0
+   ip address 1.0.0.1/32
+!
+ip routing
+!
+router bgp 1
+!!! This router ID is the same as r3
+   router-id 9.9.9.9
+   neighbor 10.14.0.4 remote-as 1
```

Comment:
Please use loopbacks to establish iBGP sessions. Add static routes to those loopbacks wherever they are referred to. The next hop should be the physical interface address of the neighbor.